static/app/views/profiling/landingAggregateFlamegraph.tsx

```diff
@@ -129,6 +168,66 @@ function AggregateFlamegraphToolbar(props: AggregateFlamegraphToolbarProps) {
   );
 }
 
+function LandingAggregaterFlamegraphFunctionBreakdown() {
+  const flamegraph = useFlamegraph();
+
+  const slowestFlamegraphFrames = useMemo(() => {
+    return getSlowestFlamegraphFrames(flamegraph).sort(sortByTotalWeight);
+  }, [flamegraph]);
+
+  const pagination = useMemoryPagination(slowestFlamegraphFrames, 20);
+
+  const paginatedSlowestFrames = useMemo(() => {
+    return slowestFlamegraphFrames.slice(pagination.start, pagination.end);
+  }, [slowestFlamegraphFrames, pagination.start, pagination.end]);
+
+  return (
+    <AggregateFlamegraphFunctionsBreakdownContainer>
+      <BreakdownTitle>{t('Functions')}</BreakdownTitle>
+      <FunctionsScrollableContainer>
+        <FunctionList>
+          {paginatedSlowestFrames.map((frame, index) => (
+            <FunctionListItem key={index}>
+              <FunctionInfo>
+                <TextOverflow>
+                  <FunctionName>{frame.frame.name}</FunctionName>
+                  {(frame.frame.package || frame.frame.module) && (
+                    <FunctionPackage>
+                      {' '}
+                      ({frame.frame.package || frame.frame.module})
+                    </FunctionPackage>
+                  )}
+                </TextOverflow>
+              </FunctionInfo>
+              <Metric>
+                {t('Samples')}: {formatAbbreviatedNumber(frame.aggregate.totalWeight)}
+                {/* {' | '}
+                {t('Instances')}: {frame.instances.length} */}
```

Comment:
Did you mean to leave this here?

Reply:
I added it, but I'm unclear if its useful to know call counts. I will likely delete it, `instances` just seems like bad wording